src/server/models/statistics/daily.ts

```diff
@@ -3,32 +3,35 @@ import Sequelize from 'sequelize'
 import { sequelize } from '../../util/sequelize'
 import { IdType } from '../../../types/server/models'
 
-export interface ClicksType extends IdType, Sequelize.Model {
+export interface DailyClicksType extends IdType, Sequelize.Model {
   readonly shortUrl: string
   readonly date: string
   readonly clicks: number
   readonly createdAt: string
   readonly updatedAt: string
 }
 
-type ClicksTypeStatic = typeof Sequelize.Model & {
-  new (values?: object, options?: Sequelize.BuildOptions): ClicksType
+type DailyClicksTypeStatic = typeof Sequelize.Model & {
+  new (values?: object, options?: Sequelize.BuildOptions): DailyClicksType
 }
 
-export const Clicks = <ClicksTypeStatic>sequelize.define('daily_stats', {
-  shortUrl: {
-    type: Sequelize.STRING,
-    primaryKey: true,
-    validate: {
-      is: /^[a-z0-9-]+$/,
+export const DailyClicks = <DailyClicksTypeStatic>sequelize.define(
+  'daily_stats',
+  {
+    shortUrl: {
+      type: Sequelize.STRING,
+      primaryKey: true,
+      validate: {
+        is: /^[a-z0-9-]+$/,
```

Comment:
can you help to check if this regex is being repeated throughout the codebase? i think we should extract it into a single shared variable

Reply:
👍  i've extracted all uses and declared the variable in our shared directory for both frontend and backend reuse